tests/ltp/patches/fix_recvmsg_recvmsg01.patch

```diff
@@ -15,7 +15,7 @@ EFAULT error behaviour is commented/disabled until github
 issue169 is fixed.
 
```

Comment:
In the original test case, the main process creates a child process
to write/read to/from sockets infinitely. SGX-LKL supports a single
process environment. The test case is modified to use a child
pthread instead of forking a child process.

One of the subtest cases is designed to test the generation of EFAULT by accessing invalid address values. Currently, SGX behavior is to call enclave abort, if the address is not within the enclave address range. Because of this test program is causing enclave abort and exiting with non-zero exit code. 

Github issue169 (https://github.com/lsds/sgx-lkl/issues/169)
is raised to fix this behavior. The subtest cases which test
EFAULT error behavior is commented/disabled until GitHub
issue169 is fixed.

Reply:
@ashokchandran, description is not modified/changed in this patch.

Reply:
done